Duplicati/Server/Runner.cs

```diff
@@ -183,18 +183,21 @@ public static IRunnerData CreateListTask(Duplicati.Server.Serialization.Interfac
                 filters);
         }
 
-        public static IRunnerData CreateRestoreTask(Duplicati.Server.Serialization.Interface.IBackup backup, string[] filters, DateTime time, string restoreTarget, bool overwrite, bool restore_permissions, bool skip_metadata)
+        public static IRunnerData CreateRestoreTask(Duplicati.Server.Serialization.Interface.IBackup backup, string[] filters, 
+                                                    DateTime time, string restoreTarget, bool overwrite, bool restore_permissions, 
+                                                    bool skip_metadata, string passphrase = null)

```

Comment:
From what I can tell, it appears there is only one usage of this method and it's been modified to provide the `passphrase`.  Should we just make this parameter required, instead of making it optional?

Reply:
Thanks, that's true. In that case, it's better to make it required.